OpenRA.Mods.Common/Scripting/Global/PaletteGlobal.cs

```diff
@@ -0,0 +1,36 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2015 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation. For more information,
+ * see COPYING.
+ */
+#endregion
+
+using System.Collections.Generic;
+using OpenRA.Mods.Common.Traits;
+using OpenRA.Scripting;
+
+namespace OpenRA.Mods.Common.Scripting
+{
+	[ScriptGlobal("Palette")]
```

Comment:
I'd prefer if this reflected the function, not the underlying mechanism. How about "Environment"? We could put all the other (global) ambient sound, weather etc. stuff into that global, too, then.